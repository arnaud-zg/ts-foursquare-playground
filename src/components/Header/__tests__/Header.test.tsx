import React from 'react'
import renderer from 'react-test-renderer'
import { Header } from '..'

describe('Component/Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header title="Title" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
