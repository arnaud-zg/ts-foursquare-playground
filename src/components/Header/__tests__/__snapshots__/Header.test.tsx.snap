// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Header renders correctly 1`] = `
<header
  className="flex justify-between items-center p-2 pl-4 pr-4 cursor-pointer select-none"
>
  <span
    className="text-indigo font-thin text-xl"
  >
    Title
  </span>
</header>
`;
